pupil-spa/src/app/services/register-input/registerInput.service.spec.ts

```diff
@@ -70,8 +80,13 @@ describe('RegisterInputService', () => {
       const event = {
         type: 'mousedown', which: 1, currentTarget: null
       };
-      registerInputService.addEntry(event);
-      registerInputService.addEntry(event);
+      const questionData = {
+        questionNumber: '1',
+        factor1: '1',
+        factor2: '12',
+      };
+      registerInputService.addEntry(event, questionData);
```

Comment:
are 2 subsequent calls intentional?

Reply:
I didn't find any particular value or reason behind having a second call so I removed it.

Reply:
thank you for spotting it